jubakit/test/test_classifier.py

```diff
@@ -86,6 +86,49 @@ def test_invalid_get_labels(self):
     # when actually iterating over it, pass it to list().
     self.assertRaises(RuntimeError, list, ds.get_labels())
 
+  def test_from(self):
```

Comment:
Please annotate this test method with ``@requireSklearn`` (as in ``test_from_matrix``); when running tests, test methods marked with the annotation will become SKIP (instead of ERROR) when sklearn is not installed.